content/neuvector-prime/azure/contents.lr

```diff
@@ -220,6 +234,34 @@ Billing will be available in Azure Portal billing
 Home > Cost Management <subscription> | Cost analysis
 
 
+## Uninstalling NeuVector and Scanner Updater
+
+Before Uninstalling NeuVector, uninstall scanner updater.
```

Comment:
add a "the" between "uninstall" and "scanner" .... "uninstall the scanner updater."